tests/test_main.py

```diff
@@ -248,7 +248,7 @@ def test_play_one_player_moves_twice(self):
         with self.assertRaisesRegexp(InvalidMovement, '"O" moves next'):
             move(self.game, self.x, position=(0, 0))
 
-    def test_play_invalid_position(self):
+    def testplay_invalid_position(self):
```

Comment:
@martinzugnoni I corrected and tried it remotely. All tests still pass. Shoud I commit/PR again?